samples/subsys/event_manager/README.rst

```diff
@@ -0,0 +1,85 @@
+.. _event_manager_sample:
+
+Event Manager
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Event Manager sample demonstrates the functionality of the :ref:`event_manager` subsystem.
+It uses an event-driven architecture, where different modules communicate through sending and processing events.
+
+
+Overview
+********
+
+The sample application consists of three modules that communicate using events:
+
+Sensor (``sensor_simulated.c``):
+  This module waits for a configuration event (which is sent by ``main.c``).
+  After receiving this event, it simulates measured data at constant intervals.
+  Every time the data is updated, the module sends the current values as measurement event.
+  When the module receives a control event from the Controller, it responds with an ACK event.
+
+Controller (``controller.c``):
+  This module waits for measurement events from the sensor.
+  Every time a measurement event is received, the module checks one of the measurement values that are transmitted as part of the event and, if the value exceeds a static threshold, sends a control event.
+
+Statistics (``stats.c``):
+  This module waits for measurement events from the sensor.
+  The module calculates and logs basic statistics about one of the measurement values that are transmitted as part of the event.
+
+
+Requirements
```

Comment:
you can remove that section as it is now supported in general (except for xtensa)